yolo_reward.py

```diff
@@ -85,12 +86,13 @@ def non_max_supression(input_bboxes: np.ndarray, iou_threshold: float = 0.50) ->
     scores = yolo_bboxes[..., 4] * torch.amax(yolo_bboxes[..., 5:], dim=-1)
 
     # Remove boxes that are below some low threshold
-    confidence_filter_mask = scores > 0.10
+    confidence_filter_mask = scores > 0.01
```

Comment:
Confidence 0.01 seems too low. The baseline is over 0.3

Reply:
My thought was that only bounding boxes above this threshold can ever teach the MLP-SAC network anything. So, it may learn some more useful correlations if it can see low probability boxes.